### PR TITLE
Actually fix python 3.5 compatibility.

### PR DESCRIPTION
**Pull request by ian-r-rose**
I got sloppy with #4962. This should be the actual fix.

### CODE REVIEW

File: jupyterlab/commands.py [hunk 1]
```diff
@@ -1769,7 +1769,7 @@ def _fetch_package_metadata(registry, name, logger):
         logger.debug('Fetching URL: %s' % (req.get_full_url()))
     try:
         with contextlib.closing(urlopen(req)) as response:
-            return json.loads(response.decode('utf-8'))
+            return json.loads(response.read().decode('utf-8'))
```
Comment (jasongrout) [inline jupyterlab/commands.py]: you could just use `json.load`, right? That's simpler.

Comment (jasongrout) [reply jupyterlab/commands.py]: never mind, that was the original code. Okay, this looks good.

Comment (ian-r-rose) [reply jupyterlab/commands.py]: Can you be more explicit? It originally was `json.load(response)`, but the response was in bytes and needed to be decoded.

**Comment by blink1073** [comment]
Thanks!

**Pull request merged**